.editorconfig

```diff
@@ -2,4 +2,4 @@
 [*]
 end_of_line = LF
 indent_style = tab
-indent_size = 4
+indent_size = 2
```

Comment:
we use tabs for indentation

Reply:
Yes I changes the indent_size only! 2 is not what it is used?

Reply:
4 is what is used.